vuepress/docs/docs/getting-started/README.md

```diff
@@ -164,6 +164,16 @@ Helm is a package manager for Kubernetes that helps you define, install, and upg
 curl -L -C - -O https://dev.entando.org/assets/yaml/entando.yaml
 ```
 
+### EntandoCompositeApp
+
+Entando uses a Kubernetes Custom Resource named `EntandoCompositeApp` that is the core of the Entando infrastructure. It's composed of 3 parts:
```

Comment:
```suggestion
To quickly deploy an application, Entando uses a Kubernetes Custom Resource named `EntandoCompositeApp`. It's composed of 3 parts:
```
Originally the EntandoCompositeApp was meant to be used for deploying the quickstart in an easy way. I'm not convinced about saying it's a `core custom resource for Entando infrastructure` . You can deploy Entando 6 even without EntandoCompositeApp.
Other custom resources are required (and therefore "core" of the infrastructure) to deploy Entando 6, like the EntandoKeycloakServer, or the EntandoApp.
What do you think?